src/main/java/com/mycompany/exercisethree/ExerciseThree.java

```diff
@@ -0,0 +1,57 @@
+package com.mycompany.exercisethree;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ *
+ * @author usuario
+ */
+public class ExerciseThree {
+
+    public static void main(String[] args) {
+        List<Double> HistoryMemory = new ArrayList<>();
+        boolean continues = true;
+        int option;
+        Scanner getInt = new Scanner(System.in);
+
+        do {
+            System.out.println("WELCOME TO THE CALCULATOR APP, PLEASE CHOOSE WHICH ONE YOU WANT TO USE ");
+            System.out.println("1. BASIC CALCULATOR");
+            System.out.println("2. ADVANCE CALCULATOR");
+            System.out.println("3. KIDS CALCULATOR");
+            System.out.println("4. HISTORY");
+            System.out.println("5. EXIT");
+            option = getInt.nextInt();
```

Comment:
Please validate value, this could lead to crashes if you enter a value different than number

Reply:
Ok, I added a try-catch to solve it.